pom.xml

```diff
@@ -3243,6 +3253,56 @@
           </dependency>
         </dependencies>
       </dependencyManagement>
+      <build>
+        <pluginManagement>
+          <plugins>
+            <plugin>
+              <groupId>net.alchim31.maven</groupId>
+              <artifactId>scala-maven-plugin</artifactId>
+              <configuration>
+                <scalaVersion>${scala.version}</scalaVersion>
+                <checkMultipleScalaVersions>true</checkMultipleScalaVersions>
```

Comment:
OK now do these settings need to be repeated? I'd have thought they don't now.

Reply:
Address a8d432f remove redundant configs, seems only Scalac args and compilerPlugins are needed

Reply:
@srowen Is there anything else that needs to be fixed? thx ~